src/frontend/src/lib/utils/network.utils.ts

```diff
@@ -45,18 +45,17 @@ export const mapNetworkIdToBitcoinNetwork = (networkId: NetworkId): BitcoinNetwo
 /**
  * Filter the tokens that either lives on the selected network or, if no network is provided, pseud Chain Fusion, then those that are not testnets.
  */
-export const filterTokensForSelectedNetwork = <T extends Token>([
-	$tokens,
-	$selectedNetwork,
-	$pseudoNetworkChainFusion
-]: [T[], Network | undefined, boolean]): T[] =>
+export const filterTokensForSelectedNetwork = <T extends Token>([$tokens, $selectedNetwork]: [
+	T[],
+	Network | undefined
+]): T[] =>
 	$tokens.filter((token) => {
 		const {
 			network: { id: networkId, env }
 		} = token;
 
 		return (
-			($pseudoNetworkChainFusion && !isTokenIcrcTestnet(token) && env !== 'testnet') ||
+			(isNullish($selectedNetwork) && !isTokenIcrcTestnet(token) && env !== 'testnet') ||
```

Comment:
Not sure about the change. `pseudoNetworkChainFusion` is a bit more readable and adding `isNullish($selectedNetwork)` here kind of duplicates the logic.

Do you plan to remove `$pseudoNetworkChainFusion` in a future PR?

Reply:
The `pseudoNetworkChainFusion` was indeed a pseudo-network selector but also included the idea that it did not contained any test network. Withouht strong types and lots of tests I think it's a risk to mix test and production tokens. Therefore, I would not rename it to `noSelectedNetwork` if this makes sense? I would even not touch it until tests are properly setup.